frontend/public/components/environment.jsx

```diff
@@ -75,10 +75,30 @@ export class EnvironmentPage extends PromiseComponent {
     }
     const envNamespace = _.get(this.props, 'obj.metadata.namespace');
 
-    this.handlePromise(Promise.all([
-      k8sGet(ConfigMapModel, null, envNamespace).catch(() => {}),
-      k8sGet(SecretModel, null, envNamespace).catch(() => {})
-    ]))
+    Promise.all([
+      k8sGet(ConfigMapModel, null, envNamespace).catch((err) => {
+        const errorMessage = err.message || 'Error, Config Maps are unavailable';
```

Comment:
I'd go with 'Could not load config maps.' or 'Could not load secrets.' as the fallback message

Reply:
nit: I'd pull the const inside the if

Reply:
yeah, good catch with the const. i refactored this a couple times. updated language.